tests/integration/KeyMappingManager/main.cpp

```diff
@@ -0,0 +1,28 @@
+#include <Core/Resources/Resources.hpp>
+#include <Core/Utils/StdFilesystem.hpp>
+#include <Gui/Utils/KeyMappingManager.hpp>
+
+#include <QtGlobal>
+
+using namespace Ra;
+using namespace Ra::Gui;
+using Idx = KeyMappingManager::KeyMappingAction;
+using Ctx = KeyMappingManager::Context;
+
+int main( int argc, char** argv ) {
+
+    /// need a base dir in argv[1]
+    if ( argc != 2 ) { return -1; }
+    std::string base {argv[1]};
+
+    QSettings settings;
```

Comment:
Maybe set organization and application ids ? https://doc.qt.io/qt-5/qsettings.html#QSettings
Note: we have default organization in `Gui`, might would make sense to use it here for consistency.

Reply:
The idea here is to have "blank" settings and skip any existing one, so I'll vote for "UnitTests" organization and settings

Reply:
Ok I push a proposal